src/Exception/DeadConnectionException.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Keboola\DbExtractor\Adapter\Exception;
 
-class DeadConnectionException extends UserRetriedException
+class DeadConnectionException extends UserException
```

Comment:
DeadConnection is retried ... but it is low level exception wrapped to `UserRetriedException`, so I had to modify the type.